Update Django versions to test

stianjensen opened the pull request:
This removes EOL python 2.7, and all non-LTS django versions that are out of supported.
Also, Django 4.0 which is now released is added to testing.

stianjensen commented:
Will probably need https://github.com/erdem/django-map-widgets/pull/127 to make tests pass on Django 4.0

stianjensen commented:
Coming back to review my own code with a big facepalm:

I've excluded all tests from running 🤦 Must have not seen the `exclude` keyword when removing the 2.7 entries...

erdem commented:
@stianjensen don't worry about your code, this project was integrated into travis-ci.org at the time that I have developed it. I already got migrating issues with travis-ci.com. I thought your changes can trigger a new build on new travis but no luck. I have reverted `.travis.yml` file and added django.4.0 env to it in the main branch. I will work on this more later.

stianjensen commented:
> @stianjensen don't worry about your code, this project was integrated into travis-ci.org at the time that I have developed it. I already got migrating issues with travis-ci.com. I thought your changes can trigger a new build on new travis but no luck. I have reverted .travis.yml file and added django.4.0 env to it in the main branch. I will work on this more later.

Okay, I see!

I created a new PR to try and cleanup, but feel free to close that again, then :+1: At this point it's probably better to just migrate to Github Actions instead, anyway.
